[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:54:32 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_e8322578bfbc484bb79dc11daff4c7a3',
    'x-envoy-upstream-service-time': '167',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=NXdam9cfFlqXJMpu8ZsqAnWBKKWyg9nEULwPkAS059M-1755010472-1.0.1.1-fesnjO2uN84GxSr._WFq4lyN_xbZPJ4xRE_en4RB0RF_8pM7ep.XrS84pI2p5JSrKZLJQWpQCdWGDZa9PqqyepdSiw0sPDDPkUwBOoM.dQo; path=/; expires=Tue, 12-Aug-25 15:24:32 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=5GhPGw7AP.BwyHUe7bLd01mAJPxu0LNdItropEU1d7g-1755010472573-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0cefb9e2f8a4e-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_e8322578bfbc484bb79dc11daff4c7a3',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:18 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_ecde79180921475da77955084fbd9054',
    'x-envoy-upstream-service-time': '76',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=VgkOaKx6CpcaDlCwLrbCM4.tjHP7dqI7D1wJzeyBL_A-1755010518-1.0.1.1-P0HURgcRj6dU7nQofKt4Aua90_WAOKSRifcuKtvIhK6MGNN6r1uh0n3dTpZYoTOPNtO8mbpyPzpJD6.DHL30ob1ggXU4Q5syJhoUfkHDAE8; path=/; expires=Tue, 12-Aug-25 15:25:18 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=iMlVxMBJuwK1I.LsqXbrzBRuJTovqKe2dPeqQ_u6ZOI-1755010518575-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d01bbce32636-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_ecde79180921475da77955084fbd9054',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:21 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_3624dd5b70584f1f93fa46e597c4fedb',
    'x-envoy-upstream-service-time': '36',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=tEEQIGuEWSdv70LHewr92wjYXjMOFDl5X.WVmGRU2Fg-1755010521-1.0.1.1-6qJTwVFO1XQkw6EOpT6T8RZHUADBrySuX7dciAdL7V_wFK4yasUMj3IL31d7ksBu1KJ4SKdX7SsMqGnkKdfz22PJPMB2aIPYvVmzuMZ.0y8; path=/; expires=Tue, 12-Aug-25 15:25:21 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=rWfhhxznqcHuXa5XztOdCexlUWnGE90XuAe5iLkuqK0-1755010521103-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d02bbef12636-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_3624dd5b70584f1f93fa46e597c4fedb',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:29 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_dea9cc8e394649dcb3bd1096c678eaaa',
    'x-envoy-upstream-service-time': '4',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=FSRFUamqfRy84JxSQn2R8O7.pTx8Tx0BbkHOCk_A29Q-1755010529-1.0.1.1-ZrDEAHd_pNRSN7Ug0o_j_5Z3a0vwSuR69Nj1MK9XVfZ.9PcArXBB14FpUSCJw.h2p230NlhGsJrjuPzI8C4KBQzVAW1tRfo50UMi3fZDbBc; path=/; expires=Tue, 12-Aug-25 15:25:29 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=pEVs4NRYVTEwdTC8QZUbO0PTbbst5hVdpix5ZLPmgng-1755010529700-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d061ceace376-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_dea9cc8e394649dcb3bd1096c678eaaa',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:32 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_b21b9ce4e21b4d8b84a4fd014853a86a',
    'x-envoy-upstream-service-time': '195',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=vZXSByq4coMB.w8CJgSy0bMLKZ.MTE934fXaIYKfers-1755010532-1.0.1.1-F3sQsm.igyv.rzfjPfLGwfjjVt4szxB8rvOk505DVgg_gCNkOqc5kEwar_EWgYUHMfu9X9hb_H_FiNYL149Q4ebNmz4yurNvmi1quDNGHzA; path=/; expires=Tue, 12-Aug-25 15:25:32 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=EqoLfd_RXp91tKc4ZvBNK4cV9urlAGYGgU0QBnTGm0c-1755010532411-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d0718cd2e376-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_b21b9ce4e21b4d8b84a4fd014853a86a',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:38 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_b4a8f0ee11f64288b980c1c57d3b2ad0',
    'x-envoy-upstream-service-time': '7',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=0VtiypCfWR0t4u6GhJnBJOTT_D5YqGJs7zDRe3Q7Z_o-1755010538-1.0.1.1-EAsZ9A_IhPeJOOo0aRYiRuV6cCDwJ4ffB8qN5tJtBwyGJR2zlGU1ZZdKcSB40C9QvXsqDqtNvDjM2kb7apOfvPm0jVqttqNaOXfqZr7fLHY; path=/; expires=Tue, 12-Aug-25 15:25:38 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=IIcthWN0porP28sBCrJ2EF8keuLkeDmx5tDSOLF5fQs-1755010538537-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d098bb46eb7e-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_b4a8f0ee11f64288b980c1c57d3b2ad0',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:40 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_837506258a854b0ba5f3f55b4c386d35',
    'x-envoy-upstream-service-time': '8',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=ufrefKxsX3YEvU7zXFVUecpSzyjcXyvcbArJBjxSDW8-1755010540-1.0.1.1-u6pHP7kvhlFqJ5YLf8k0Cws0RBBfXXRgwERaOA2nZ8rMfNsAwrBhd56E56oANK1ICMvaEcPW70Ijzcea07hVWipOTg5MpznuX1GAgH.LVmM; path=/; expires=Tue, 12-Aug-25 15:25:40 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=_aTG4QJiUxtMLuCJhgvUBL9HLgNXBLN4pM9q5ImYNWo-1755010540935-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d0a7ac20eb7e-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_837506258a854b0ba5f3f55b4c386d35',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:46 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_e6f5658c39c847d38891a2e68d47b877',
    'x-envoy-upstream-service-time': '6',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=V8sB.R98VYxgBH.EcEkVqqPb.7rlRkeTIHAVVoTHpKQ-1755010546-1.0.1.1-fN9ltRHposKEBqo6JOifpSNX6vW7RQw_FSF6shcdC1xnagQR.zHQbyn15IV6Lq6DWaxkF2uuUKLeiJO.od4HDCjIJQv.aX.PaXu8VaFpJrU; path=/; expires=Tue, 12-Aug-25 15:25:46 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=y8cIBTqSkU6vqgcipUsUiR5z84R2KOu8ZXwc4PaHnok-1755010546608-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d0cb1af5eb7e-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_e6f5658c39c847d38891a2e68d47b877',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:49 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_8a1e4904d0f84e58bd5f9b707cc592d3',
    'x-envoy-upstream-service-time': '16',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=0qYXPRc3Vabv7appGweUIcrZSUjdOB0b5ArgYFVh9b0-1755010549-1.0.1.1-IdiKoSDY0pMCik8I8f6Z1xivsabcV9ne.B7eise4deSyTuVVNNkFbYRl74W.zikkQQxwozL6ZpE7Q58Noy9Gsjaw1_rfghEM2x_0vGUma38; path=/; expires=Tue, 12-Aug-25 15:25:49 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=uaZQtk1RJZzWy5s7YbGAvLPr1hevGKlPcpb8Grsjwqk-1755010549127-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d0dafd70eb7e-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_8a1e4904d0f84e58bd5f9b707cc592d3',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:55:57 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_82635e12888e48038ba1b0384417fa05',
    'x-envoy-upstream-service-time': '4',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=gkAFqmiGyQQj9lP2hljNl_KKmX00whXIPdoUFYoYstA-1755010557-1.0.1.1-Ze4W.suGgFbS018HnBzILiwETqVxiIhAT3DgjwNFB5Sbb0ILovq34n.kBUezpl9t633cWBAFV8svgeBMpW8CeVDL1Tb8dsSAhbeKyKbzlbw; path=/; expires=Tue, 12-Aug-25 15:25:57 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=cCQKpbmsMW4erI3xFo29G_ACz9b65O3gCeskwbNoZRw-1755010557934-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d1122c436872-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_82635e12888e48038ba1b0384417fa05',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:56:00 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_a3ab8e20402c4f5ab12aa5fa47d573a9',
    'x-envoy-upstream-service-time': '7',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=AefZk4jj0C0SHuO9KGSnTRHCM0BiA32j_rpq6XFxX8s-1755010560-1.0.1.1-sJqAU7ZqFCnVMkubJObWqxylHcSml4EFFjS0OZIv6NLOQ7lA1Bp9B_7LgFP8gAbBxyHINLScxgbdAPu50rsKYv72eQf1F6c5N97_Nxjneec; path=/; expires=Tue, 12-Aug-25 15:26:00 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=Bmv3W.nmIi5FwZzBXAc4K3wQsl03X8SqtL28ZNlcD4I-1755010560426-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d121ad396872-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_a3ab8e20402c4f5ab12aa5fa47d573a9',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:56:08 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_bb06485130094658ab3611f995c25bf7',
    'x-envoy-upstream-service-time': '15',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=dPQLIuJGmAP6M9am.rTiZE2_xyK3PVWpE6Tfw5w49VQ-1755010568-1.0.1.1-Sc.iizpU4jtIWJuaWnrIR1RsQ2WHob.PhnvP1RsIm8s3JEb8WV1surl9u_D7S7fTa_15Rmy9Bs6qeL4Th.h098w6m_S9QL5qej_6wXa72z4; path=/; expires=Tue, 12-Aug-25 15:26:08 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=6aNNyzCPB_DPP4gRSQtexSaj7YFeXQYSFsqPl2G2AVE-1755010568230-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d1526f97f5e5-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_bb06485130094658ab3611f995c25bf7',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:56:10 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_7632668b7ef7461088a6bfd29e7ca3f3',
    'x-envoy-upstream-service-time': '7',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=IDM70mnuw6oHKkc.BDxG5KSKeGWCP0uDYX5HWAy9F0E-1755010570-1.0.1.1-STEHsNKTtiNlJio39ti7ZqQpqPMwZpv0dO9F.iT_5Ao1V4q85Dd2fOG36cbPzYkaIMgf8WhTF4Yas_82fGvcfo5Wxs1Iiu0DOGOsYBHNqYM; path=/; expires=Tue, 12-Aug-25 15:26:10 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=CJEnTNNYycfVhkIFz5k70wtjbYMYQ4Tu8.ZPr781gzM-1755010570575-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d1610805f5e5-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_7632668b7ef7461088a6bfd29e7ca3f3',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:58:21 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_8d632aa2ab424f8e9db8d18b469d59d6',
    'x-envoy-upstream-service-time': '8',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=Art7eRtnBlZO3UvNF1rGbuP9jqt0soB98Jd4aVIyQRg-1755010701-1.0.1.1-Ik7evKxI1w7Rmcl8ic6xHkrXyrkfcqGA4KwHCI4yBLoQMhiLDE.bQ9e.6.l2wrUXaom2rAbYZb9uvIRiTnfDpaLz6SOyZr6VlsMI_bsBIgE; path=/; expires=Tue, 12-Aug-25 15:28:21 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=ELCEF2dwwi6A6O_BIDh.2BVLG54aYgC0WTXaGhHNomA-1755010701509-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d4939c46263e-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_8d632aa2ab424f8e9db8d18b469d59d6',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:58:23 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_04e48cb707ec44dba08fca5a6745fe63',
    'x-envoy-upstream-service-time': '3',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=WHeNdMsliUf4MZEWfYHscQSAWdbwbKmzZG3QInNKvOs-1755010703-1.0.1.1-d4hr4oj9yPhyOlwGET894c1lXZkokGqIO0fedp_JThoOBfAFx1l6FV0x9SNcli2f7kN04Fxy5JFpXleUgiaFxXIK2YF9SWGlP7RXf0q1kPc; path=/; expires=Tue, 12-Aug-25 15:28:23 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=5GCd29ABV6ckMSUavN.zWbyZ4UjBvFHlawTMjv8AO5U-1755010703993-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d4a31eba263e-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_04e48cb707ec44dba08fca5a6745fe63',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:58:32 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_d6455198cd5a4fd1a86335a4d9970b2b',
    'x-envoy-upstream-service-time': '5',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=W2RCdHVNb2A0QuekRCnQ3iutoxjcJtyiSJ9xhDYMtdc-1755010712-1.0.1.1-tkZzVXBcgXtw3R9qmTtvhJfPz42Fn.PZRCfQXhS_jAVbHOOn3ygRVm1K68UATAXJk_89oW1Fwtgvv.BmS8_v648iRJonlgY0FRvmlbaoiCo; path=/; expires=Tue, 12-Aug-25 15:28:32 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=93yJoClfd6VGnd80IdriZFet102CNQ8gZYSGBqrcIeg-1755010712204-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d4d65bf4d74f-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_d6455198cd5a4fd1a86335a4d9970b2b',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:58:34 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_27be286eaa6540e5950ee2829292fd41',
    'x-envoy-upstream-service-time': '16',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=L8YHmW0YeveZSX6JwAyvvqrj7xZOdfvdWJB62HQswGA-1755010714-1.0.1.1-dgwzWn_uTNZyA82BRN2W1I11lXTQlye_Fg76_AwirdqcMWuxXIvlTshTsJsmMUi4PqR5mO61EPQRQCnsH7UznsgkxnREQgzHfuS1tmTiW9o; path=/; expires=Tue, 12-Aug-25 15:28:34 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=ECL5WFOPTj5wKNIX3K_3d0yRQN6SYkeQZb2mETUbH_Y-1755010714686-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d4e5cb91d74f-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_27be286eaa6540e5950ee2829292fd41',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:58:41 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_5eb5c70fe18b4dd3849ed4a02b6cb77f',
    'x-envoy-upstream-service-time': '6',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=1labQRTdjdiiIOWyVd8FMK2VxjNlWhPx3Ih_AHeS3gc-1755010721-1.0.1.1-DGTlTjSoHw6uGzVhQuSR8GHHxoKQ3tzWZgDIirlA53HlRV.4Aka9wXD7YHLkp3tzwKsY.DUywpmZfILdhwwINntKYljHe9jGbOnagdGXTSc; path=/; expires=Tue, 12-Aug-25 15:28:41 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=Qe_6yLe_Ypg0ekXH9JP6Dol9DITU40Ez_8lrgfw9Vrs-1755010721043-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d50d7e3a7364-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_5eb5c70fe18b4dd3849ed4a02b6cb77f',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:58:43 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_c45fa6e170cb4aa7bb764b8e35684cac',
    'x-envoy-upstream-service-time': '4',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=_qmMyb4MzFB9Bin_07bO7ShpN04sBUWsg3Tv.OiW.gg-1755010723-1.0.1.1-RTrRkruoBC.1QeTZ7vxBflWTet1pphtx1fAYx4T8z4mQK2Ef16SUcb4D4yHmHtnk1XiTcZOjHqihKRA7DGVN4rXiApWXyqdJ_ETmowXqDaU; path=/; expires=Tue, 12-Aug-25 15:28:43 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=O73x3GzTa_eQpdREkxQHzhZDvxLfBMxFb6JbRyGbgZ8-1755010723588-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d51d08467364-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_c45fa6e170cb4aa7bb764b8e35684cac',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:58:49 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_143af2db39344ad6976643da4ecca872',
    'x-envoy-upstream-service-time': '3',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=zP37juAsjNqJT4dQnhe6yCQBzAmaDJvV9altSUOvXgM-1755010729-1.0.1.1-hf9e_AprBqhsiMGJuLx.fZKy3ABEGnnmkoeIByvmi9uGSm4GhWj1aborEQuH2ZwQh3ic4AG7h9._V.mRoPE0vkMk0pOz9QvJryQmuQpPFIg; path=/; expires=Tue, 12-Aug-25 15:28:49 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=E2hWEe_dj37GSZxTHGw4yoACJHdelw0b07R4dU3IdXY-1755010729451-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d5424c507364-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_143af2db39344ad6976643da4ecca872',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:58:51 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_1c64e0136a454191926157c01950f01c',
    'x-envoy-upstream-service-time': '8',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=rc5yEiyWL0.H6calz8XBSJ2fwnyqCm5Le_kOpWg8S1w-1755010731-1.0.1.1-0UtLvM2YVoVvfw7Bh.kbP47csAtlErpzBC5wLfNsVPPQI0.3S5jj04BimN1sSOfdQhE7qpTo64l8AknKRC6o_ZBx9W2egDmBFhdjK7yTBng; path=/; expires=Tue, 12-Aug-25 15:28:51 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=yOAB8zA_HIMPB.7o4pSh0sFQ2XWFnBKgDyjhKIEaIAI-1755010731928-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d5517ce37364-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_1c64e0136a454191926157c01950f01c',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:59:00 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_197ed92953d741789b52d002b01bfc90',
    'x-envoy-upstream-service-time': '4',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=7hY0vDQLQwdCcmQLtdT7AfT60Q3.vURZ3432ujKWxr4-1755010740-1.0.1.1-HOczrLe4NBPApNbF1mjNAmhYhoU.j_.ZRLD_1YFjcSl0R.oC2HLyGaApw726zQC7bk1zaLxKFdADf45mh1qlz5qzyQ.CTRW7WFPAhW4h43o; path=/; expires=Tue, 12-Aug-25 15:29:00 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=ZvlfspEfivECUBK992hSOvA_0gojDwkgQ7i87Ltqb30-1755010740695-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d5888c0d7969-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_197ed92953d741789b52d002b01bfc90',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:59:03 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_1bdbbe68432844e590c8b9af88e0321a',
    'x-envoy-upstream-service-time': '10',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=P1cmNbNjJo.5I7d4WFJXQ.Dvoz7YcDwEeSA2uPW70Ss-1755010743-1.0.1.1-dtOyr71mU6tL_2JnoAZgsz4oGCv16vdxg2v8tiL8u_cqQAw09SFlz45pUrVTPGN.m3TEzJAgJjEaaN.fgFElLGG1KOFvgpj6bMu9s3YonY4; path=/; expires=Tue, 12-Aug-25 15:29:03 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=G5ArXrv_RvglOonzo7atR3lqJkv2nzMR9L.JfgrGgEQ-1755010743267-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d59879527969-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_1bdbbe68432844e590c8b9af88e0321a',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:59:10 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_c634027ba6a543d095456e27edd1dd86',
    'x-envoy-upstream-service-time': '4',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=x8DwJP2FrRpMPKyottNfQEWDVprEx6ueHJnHAXNj_W4-1755010750-1.0.1.1-zOaoBevjwz_4YzmrLaQzqviMAaUjVgdVmaG8yFv.EkUOkCEnifUriNG28082n2nDNo6SX1NjyVt.RGvqXeathjOEdLSbZWs0SyJ8rAytgd0; path=/; expires=Tue, 12-Aug-25 15:29:10 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=hjoTZa62jekAhV0YQM8iWah5u_rXIVq18aeHHba7DFU-1755010750981-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d5c8bb2b8a66-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_c634027ba6a543d095456e27edd1dd86',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
[/api/tts] error: Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
    at APIError.generate (../src/core/error.ts:96:13)
    at OpenAI.makeStatusError (src/client.ts:445:27)
    at OpenAI.makeRequest (src/client.ts:668:23)
    at async POST (src\app\api\tts\route.ts:32:17)
  30 |     // OpenAI の TTS（テキスト読み上げ）
  31 |     // よく使われる音声名: "alloy" ほか。使えない場合は "alloy" に戻してください。
> 32 |     const resp = await openai.audio.speech.create({
     |                 ^
  33 |       model: "gpt-4o-mini-tts",
  34 |       voice,
  35 |       input: text, {
  status: 429,
  headers: Headers {
    date: 'Tue, 12 Aug 2025 14:59:13 GMT',
    'content-type': 'application/json; charset=utf-8',
    'content-length': '337',
    connection: 'keep-alive',
    vary: 'Origin',
    'x-request-id': 'req_8fc707d9476c43da83f1543bc4abb274',
    'x-envoy-upstream-service-time': '10',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=8gE20kqrC5tfRLHH5cxdD8QJmYZYWGzUocg8MTuoAeI-1755010753-1.0.1.1-WZVzzeGCA3wU_sDR7nNW2gxRgJ8SEgmZK47cJsCiy46lijfuRtDxnrrXJxI_K6iNvlTDuSN95MCPBk.Q5Cro6sdClBpNenp99DblJQxG6ZE; path=/; expires=Tue, 12-Aug-25 15:29:13 GMT; domain=.api.openai.com; HttpOnly; Secure; SameSite=None, _cfuvid=un5JMyUonRolmC4NblEAU7Gh.gAYFrho2kbCf5s7Ej0-1755010753436-0.0.1.1-604800000; path=/; domain=.api.openai.com; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=31536000; includeSubDomains; preload',
    'x-content-type-options': 'nosniff',
    server: 'cloudflare',
    'cf-ray': '96e0d5d7fb138a66-NRT',
    'alt-svc': 'h3=":443"; ma=86400'
  },
  requestID: 'req_8fc707d9476c43da83f1543bc4abb274',
  error: [Object],
  code: 'insufficient_quota',
  param: null,
  type: 'insufficient_quota'
}
^C^C